draft-fossati-seat-expat.md

```diff
@@ -168,6 +168,19 @@ To maintain a cryptographic binding between the Evidence and the authentication
 - The Evidence is specific to the authentication event and cannot be replayed across different TLS sessions.
 - The Evidence remains tied to the cryptographic context of the TLS session.
 
+## Cryptographic Binding of the Evidence to the TLS Session
+
+The attester MUST bind the attestation evidence to the active TLS session. To do so, the attester derives a
+binding value using the TLS exporter and the exporter_master_secret of the current TLS connection. The exporter
+invocation uses:
+
+* the label "Attestation Binding", and
+* the certificate_request_context from the CertificateRequest message as the exporter context.
+
+The attester MUST include the exporter value exactly as produced in the attestation evidence.
+
+To allow verification, the TLS endpoint that receives the attestation evidence MUST compute the exporter value using the same exporter invocation described for the attester. The TLS endpoint can verify the exporter binding directly; if it does, it MUST reject the attestation evidence when the values do not match. If the TLS endpoint does not perform this check itself, it MUST convey the computed exporter value to the Relying Party (RP) so that the RP can perform the comparison as part of attestation validation.
```

Comment:
> If the TLS endpoint does not perform this check itself, it MUST convey the computed exporter value to the Relying Party (RP) so that the RP can perform the comparison as part of attestation validation.

Confused: isn't the TLS endpoint here the same entity as the RP? Also, I'm not sure it's secure to convey the exporter (to a Verifier) as-is, you probably want one more round of hashing.

Reply:
I meant the verifier which validates the attestation evidence, replaced RP with "verifier". The exporter value is hashed based on the inputs 
a) label which is set to "Attestation Binding" 
b) context is set to certificate_request_context 
c) exporter_master_secret
Because this value is the output of a one-way hash function, it cannot be used to recover the exporter_master_secret.